packages/api/src/routes/webhook/fitbit.ts

```diff
@@ -0,0 +1,51 @@
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import status from "http-status";
+import { processData } from "../../command/webhook/fitbit";
+import { Config } from "../../shared/config";
+import { asyncHandler } from "../util";
+
+const routes = Router();
+
+/** ---------------------------------------------------------------------------
+ * GET /webhook/fitbit
+ *
+ * Verify Fitbit WH subscriber. Receives a verification notification and sends back a verification post request.
+ *
+ */
+routes.get(
+  "/",
+  asyncHandler(async (req: Request, res: Response) => {
+    if (req.query.verify) {
+      console.log(
+        "Verifying the subscriber. Make sure to use the verification code provided in the dev.fitbit dashboard!"
+      );
+
+      if (req.query.verify === Config.getFitbitVerificationCode()) {
+        console.log("Received correct verification code.");
+        return res.sendStatus(status.NO_CONTENT);
+      } else {
+        console.log("Incorrect verification code detected!");
+        return res.sendStatus(status.NOT_FOUND);
+      }
+    } else {
+      return res.sendStatus(status.OK);
```

Comment:
Sanity check: do we want to send OK when no `verify` param is provided? And not just `NO_CONTENT`, but `OK`?

Reply:
Changed to status.NOT_FOUND with an accompanying error message

Reply:
Sounds good. Took this to this [other thread](https://github.com/metriport/metriport/pull/664#discussion_r1265887560) to reference the new code.